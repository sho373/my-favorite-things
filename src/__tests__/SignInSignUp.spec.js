import React from 'react';
import SignIn from '../pages/sign-in/sign-in.component';
import SignUp from '../pages/sign-up/sign-up.component';
import { shallow } from 'enzyme';
import {configure} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({adapter: new Adapter()});

it('should render SignIn page component', () => {
    expect(shallow(<SignIn />)).toMatchSnapshot();
});

it('should render SignUp page component', () => {
    expect(shallow(<SignUp />)).toMatchSnapshot();
});