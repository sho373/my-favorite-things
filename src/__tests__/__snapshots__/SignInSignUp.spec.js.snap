// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render SignIn page component 1`] = `ShallowWrapper {}`;

exports[`should render SignUp page component 1`] = `ShallowWrapper {}`;
